examples/complete-vpc/main.tf

```diff
@@ -1,14 +1,15 @@
 provider "aws" {
-  region = "eu-west-1"
+  region = local.region
 }
 
 locals {
-  name   = "complete-example"
+  name   = "ex-${replace(basename(path.cwd), "_", "-")}"
```

Comment:
All of the provider/locals/tags updates were my fault - just aligning the examples to be consistent